docs/styleguide/styles/_variables.scss

```diff
@@ -0,0 +1,24 @@
+//all variable related to Web site techne.yaas.io and techne library overwrites should be located here
+
+//size
+$primary-nav--height: 50px;
+
+//fonts
+$font-family-dosis: 'Dosis', sans-serif;
+$font-size-base:          14px;
+$font-size--footer: ceil(($font-size-base * 1.07));
```

Comment:
Just put this in `footer.scss` if this variable will not be used anywhere else. And why not just make it `15px`? Being a global nav (across YaaS), it makes sense to hardcode these values.